docs/self-managed/console-deployment/configuration.md

```diff
@@ -9,6 +9,9 @@ description: "Read details on the configuration variables of Console Self-Manage
 Console Self-Managed is available only to [Enterprise customers](/reference/licenses.md#console).
 :::
 
+As a Spring Boot application, Console supports any standard
```

Comment:
I'm not sure about this. I remember that Console doesn't support the Spring convention out of the box, only what we agreed on.